### ISSUE TITLE
Provide support and documentation for 1000 genomes references

### ISSUE THREAD

**Issue by jeromekelleher** [ga4gh/server#1019]
To support serving the 1000 genomes references we will need to make it possible/easy to import the references used by the 1000 genomes BAMs and VCFs. This should be as easy as 
```
ga4gh_repo add-referenceset <repodir> <url> --name=GRCH38_plus_decoy
```

Here is an one 1000 genomes reference:

ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/technical/reference/GRCh38_reference_genome/GRCh38_full_analysis_set_plus_decoy_hla.fa 

See also the README in this directory.

Because we can't support remote access on FASTA files (as far as I know), I think we have to download the full FASTA file into the data repo.

Changes required:

- [ ] 1. Repo manager should support downloading a file from a URL and copying it into the data repo tree (this functionality will also be required for BAM/VCF index files). If an index exists alongside the specified reference, download it and use it. If not, create a new one using pysam.
- [ ] 2. We change the structure of ReferenceSets so that we use a single indexed fasta file (this should be addressed in a separate issue/PR before tackling this issue).
- [ ] 3. Repo manager should add CLI options to support setting the blank metadata fields in the Reference and ReferenceSet objects that we cannot infer. (Also probably best done in a separate issue/PR).

The goal of this issue is to make clear the steps we need to take to support 1000 genomes references. This is a prerequisite for supporting the import of 1000 Genomes BAMs/VCFs from a URL because we will need to have the right references imported into our repository before we can import reads and variants.

**Comment by david4096** [comment]
Remote files in repo manager https://github.com/ga4gh/server/issues/970

**Comment by jeromekelleher** [comment]
Thanks @david4096! Let's make #955 the first step in this process.

**Comment by jeromekelleher** [comment]
Closed by #1166.
